stock_picking_line_sequence/models/stock_move.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2017 Camptocamp SA - Damien Crier, Alexandre Fayolle
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo import api, fields, models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    # re-defines the field to change the default
+    sequence = fields.Integer("HiddenSequence", default=9999)
+
+    # displays sequence on the stock moves
+    visible_sequence = fields.Integer(
+        "Sequence",
+        help="Shows the sequence in the Stock Move.",
+        related="sequence",
+        readonly=True,
+        store=True,
+    )
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        """(re)initialize the sequences of the moves within a picking"""
```

Comment:
Do not change create docstring
```suggestion
        # (re)initialize the sequences of the moves within a picking"""
```

Reply:
Isn't a docstring the best way to document what a method (or an override) does?

Reply:
A method yes. An override no as it's overriding the docstring of what create does

Reply:
I must admit that it is out of intuition that I have been doing this, but I have been doing this for over a decade. Do you have a pointer to an Odoo or OCA guideline about this?

Reply:
No I don't. But you can check if my saying is right by opening a shell and running `help(env["stock.move"].create)`

Reply:
But if I pull the documentation of an Odoo module with a sphinx-like tool, then what I'd want to see is what the override changes to the behaviour rather than the docstring of the super method.

Reply:
I don't have any recommendation on that. But, for my perspective, normal comments (`#`) are not displayed in IDE's helper (vscode). Only docstrings `"""`.

Reply:
@rousseldenis Do you agree with the suggestion? What would give the best usability in visual IDEs (that I don't use)?

Reply:
@StefanRijnhart yes, your suggestion is the good one.

Reply:
@rousseldenis you mean the one with docstring or comment?

By the way, it's not an override (where docstring would be right) but an extend (where it's debatable)

Reply:
@rousseldenis Indeed, I was talking about jbaudoux' suggestion to replace the docstring by a comment.

Reply:
OK, done.